### PR TITLE
Creates virtual env

### PR DESCRIPTION
**Pull request by sungeunbae**
Testing

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -43,8 +40,8 @@ pipeline {
                     echo "[ Python used ] : " `which python`
                     cd ${env.WORKSPACE}
                     echo "[ Installing ${env.JOB_NAME} ]"
-# full installation is not possible as it takes more than 3.0Gb for building
-#                    python setup.py install
+# full installation is not possible as it takes more than 3.0Gb for building and kills the server
+#                   python setup.py install
                     python setup.py build_ext --inplace
 		            python konno_setup.py
```
Comment (jasonmotha) [inline Jenkinsfile:46]: That's odd that you can create the konno matrices manually but not from the setup.py script?

File: requirements.txt [hunk 2]
```diff
@@ -1,9 +1,10 @@
 pytest>=3.5.0
 pytest-black
-numpy
+numpy==1.20
```
Comment (jasonmotha) [inline requirements.txt:3]: If you are using isclose do you have to hardcode the numpy version?

Comment (sungeunbae) [reply requirements.txt:3]: Good catch

File: Jenkinsfile (hunk 1 above)
Comment (sungeunbae) [reply Jenkinsfile:46]: konno_setup.py has a limited depth (num_to_gen=5), just enough for auto testing. 
Konno matrices built by setup.y runs in at memory alloc failure (2.0Gib) at 16385

File: requirements.txt (hunk 2 above)
Comment (sungeunbae) [reply requirements.txt]: BTW, fixing numpy==1.20 was not good enough to produce the result giving the same number of significant digits. Finding out what exact package was causing this was going to be a treasure hunt promising little prize..

**Review by jasonmotha — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

**Comment by sungeunbae** [comment]
Had to re-introduce numpy==1.20 as numba doesn't like the latest numpy

**Review by jpa198 — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**